packages/core/ios/Sources/DdSdk.swift

```diff
@@ -213,6 +213,15 @@ class RNDdSdk: NSObject {
             _ = ddConfigBuilder.enableCrashReporting(using: DDCrashReportingPlugin())
         }
 
+        _ = ddConfigBuilder.setRUMResourceEventMapper({ resourceEvent in
+            if ((resourceEvent.context?.contextInfo.contains(where: { (key: String, value: Encodable) in
+                key == InternalConfigurationAttributes.dropResource
+            })) ?? false == true) {
```

Comment:
I'm not sure you need this `?? false == true`, `== true` should be good enough as `nil` will not equal true.

Actually I think you might be able to do:
```swift
if resourceEvent.context?.contextInfo[InternalConfigurationAttributes.dropResource] != nil {
}
```

As the default for a dictionary that does not contain a key in Swift is to return nil.

Reply:
Oh yes it's much better, thanks a lot!